### PR TITLE
Removed obsolete sumdbaudit command

### PR DESCRIPTION
**Pull request by mhutchinson**
This has been replaced with sumdbverify, which now has useful information that was in the sumdbaudit README incorporated

### CODE REVIEW

**Comment by mhutchinson** [comment]
I hadn't realized that these tools had some entanglement. Switching to draft until I've bottomed this out.

File: clone/cmd/sumdbclone/internal/client/sumdb.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+func (c *SumDBClient) tilePath(offset int) string {
+	nStr := fmt.Sprintf("%03d", offset%pathBase)
+	for offset >= pathBase {
+		offset /= pathBase
+		nStr = fmt.Sprintf("x%03d/%s", offset%pathBase, nStr)
+	}
+	return nStr
+}
+
+func dataToLeaves(data []byte) [][]byte {
```
Comment (AlCutter) [inline clone/cmd/sumdbclone/internal/client/sumdb.go:123]: Is this the same as `bytes.Split(data, []byte{'\n'})`?

File: clone/cmd/sumdbclone/internal/client/sumdb.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	}
+	defer func() {
+		if err := resp.Body.Close(); err != nil {
+			glog.Errorf("resp.Body.Close(): %v", err)
+		}
+	}()
+	if resp.StatusCode != 200 {
+		return nil, fmt.Errorf("GET %v: %v", target, resp.Status)
+	}
+	data, err := io.ReadAll(io.LimitReader(resp.Body, 1<<20))
```
Comment (AlCutter) [inline clone/cmd/sumdbclone/internal/client/sumdb.go:159]: could this just directly `return io.ReadAll(...)` since you're not decorating the error?

**Review by AlCutter — APPROVED** [review]
Just a couple of little suggestions in what (now) looks like pre-existing code - the link I clicked on to review this brought me to just the 2nd commit, and the entirety of those re-added files appeared to be new.

File: clone/cmd/sumdbclone/internal/client/sumdb.go (hunk 2 above)
Comment (mhutchinson) [reply clone/cmd/sumdbclone/internal/client/sumdb.go]: Done.

File: clone/cmd/sumdbclone/internal/client/sumdb.go (hunk 1 above)
Comment (mhutchinson) [reply clone/cmd/sumdbclone/internal/client/sumdb.go:124]: No, but I've replaced it with something similar. Annoying to split on "\n\n" and then need to re-add it, but I'll survive.

**Pull request merged**